src/Parameters/CreateMeetingParameters.php

```diff
@@ -31,780 +99,314 @@ class CreateMeetingParameters extends MetaParameters
     /**
      * @var string
      */
-    private $meetingId;
+    protected $name;
 
     /**
      * @var string
      */
-    private $meetingName;
+    protected $meetingID;
 
     /**
      * @var string
      */
-    private $attendeePassword;
+    protected $attendeePW;
 
     /**
      * @var string
      */
-    private $moderatorPassword;
+    protected $moderatorPW;
 
     /**
      * @var string
      */
-    private $dialNumber;
-
-    /**
-     * @var int
-     */
-    private $voiceBridge;
+    protected $welcome;
 
     /**
      * @var string
      */
-    private $webVoice;
+    protected $dialNumber;
 
     /**
      * @var string
      */
-    private $logoutUrl;
+    protected $voiceBridge;
 
     /**
      * @var int
      */
-    private $maxParticipants;
+    protected $maxParticipants;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $record;
+    protected $logoutUrl;
```

Comment:
The correct api param is logoutURL not logoutUrl

Reply:
Done.